client/src/components/Donation/components/DonateText.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+
+class DonateText extends Component {
+  render() {
+    return (
+      <div className='text-center'>
+        <p>
+          freeCodeCamp.org is a tiny nonprofit that's helping millions of
+          people learn to code for free.
+        </p>
+        <p>
+          Join <strong>4,180</strong> supporters.
```

Comment:
There is a component here that does some calculations on this number here:

`client/src/components/Supporters.js`

We use it on the `/welcome` page, the number could probably be extracted out and used here also.